client/src/components/MaterialsFilters/utils/initFilters.ts

```diff
@@ -0,0 +1,81 @@
+import isValidInteger from '@/utils/isValidInteger';
+
+type QueryData = {
+    park?: string,
+    category?: string,
+    subCategory?: string,
+    tags?: string,
+};
+
+type MaterialsFilters = {
+    park: string,
+    category: string,
+    subCategory: string,
+    tags: string[],
+};
+
+type MaterialsFiltersParams = {
+    park?: number,
+    category?: number,
+    subCategory?: number,
+    tags?: string[],
+};
+
+/**
+ * Initialise les filtres pour les champs de filtrage du matériel
+ *
+ * @param query L'objet `query` de la route actuelle
+ * @returns Un objet correctement formaté pour les filtres
+ */
```

Comment:
J'ai tendance à ajouter des points et utiliser `@return` plutôt que `@return`:

```suggestion
/**
 * Initialise les filtres pour les champs de filtrage du matériel.
 *
 * @param query L'objet `query` de la route actuelle.
 *
 * @return Un objet correctement formaté pour les filtres.
 */
```

Reply:
Ah OK, pour le `@returns` (avec un s), c'est mon IDE qui l'a écrit tout seul via un snippet... :sweat_smile: Mais je prend note !